src/main/java/frc/trigon/robot/commands/Commands.java

```diff
@@ -70,4 +75,12 @@ public static Command duplicate(Command command) {
                 command.getRequirements().toArray(Subsystem[]::new)
         );
     }
+
+    public static Command getNoteCollectionCommand() {
+        return new ParallelCommandGroup(
+                new AlignToNoteCommand().onlyIf(() -> CommandConstants.SHOULD_ALIGN_TO_NOTE),
+                LEDStripCommands.getStaticColorCommand(Color.RED, LEDStripConstants.LED_STRIPS).asProxy().onlyIf(() -> CommandConstants.SHOULD_ALIGN_TO_NOTE),
```

Comment:
Do orange LEDs only if you are NOT aligning to the note, to not conflict with the alignment LEDs